src/webhook/mutation/pod_mutator.go

```diff
@@ -687,7 +695,7 @@ func updateInstallContainerOneAgent(ic *corev1.Container, number int, name strin
 }
 
 // updateContainerOA sets missing preload Variables
-func updateContainerOneAgent(c *corev1.Container, dk *dynatracev1beta1.DynaKube, pod *corev1.Pod, deploymentMetadata *deploymentmetadata.DeploymentMetadata) {
+func (m *podMutator) updateContainerOneAgent(c *corev1.Container, dk *dynatracev1beta1.DynaKube, pod *corev1.Pod, deploymentMetadata *deploymentmetadata.DeploymentMetadata, nsName string) {
```

Comment:
Since another parameter has been added, and the signature was too long before already, please refactor your code such that parameter list does not have more than two parameters.

Reply:
I've done a slight refactor which sort of proliferated throughout the pod_mutator code. I've added two new fields to the `podMutator` struct for the `currentPodNS` and also for  the current/active `dynaKube` instance related to the incoming pod request. 

Doing this in turn shaved two args from the function declaration. Please review.